adhocracy4/ratings/static/ratings/react_ratings.jsx

```diff
@@ -23,8 +25,10 @@ var RatingBox = React.createClass({
   },
   handleRatingModify: function (number, id) {
     api.rating.change({
-      object_pk: this.props.objectId,
-      content_type: this.props.contentType,
+      urlReplaces: {
```

Comment:
Okay I could agree that before they were passed as data while beeing used in the url. The client has already an additional parameter for `id` which also goes into the url. So why not stop passing any url parts as data then, and instead pass them as additional url paramter.

```
api.rating.change(
   {'id': id, 'object_pk': object_pk, 'content_type': content_type},
   {'value': value}
)
```

While at it you could probably also specify the url more directly. Also to reduce the complexity and functionality in the api client.

```
api.change(
   ['contentTypes', content_type, 'objects', object_pk, 'ratings', id],
   {'value': value}
)
```